### PR TITLE
Sidebar - WIP

### PR DESCRIPTION
**Pull request by kelfro**
# Description

Creation of the navbar that is used on each page (except for the landing), and added to all current pages.

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change status

- [x] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: frontend/src/pages/Houses.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, { useState, useLayoutEffect, useRef } from 'react';
```
Comment (NandoTheessen) [inline frontend/src/pages/Houses.tsx]: This whole file now lives in pages/Houses/, if we were to merge this, we'd all of a sudden have 2 of them with this being the older version.

Comment (NandoTheessen) [reply frontend/src/pages/Houses.tsx]: This change has already landed on master, so please git pull and remove this if necessary

Comment (kelfro) [reply frontend/src/pages/Houses.tsx]: I did a pull from master and everything seems to be updated to the latest version.

**Comment by honda0306** [comment]
New PR opened at 
https://github.com/Lambda-School-Labs/labs9-cleaner-pos/pull/50

**Pull request closed without merging**